prov/rxm/src/rxm.h

```diff
@@ -421,10 +421,50 @@ struct rxm_conn {
 	struct dlist_entry posted_rx_list;
 	struct dlist_entry sar_rx_msg_list;
 	struct util_cmap_handle handle;
-	/* This is saved MSG EP fid, that hasn't been closed during
-	 * handling of CONN_RECV in CMAP_CONNREQ_SENT for passive side */
-	struct fid_ep *saved_msg_ep;
-	struct dlist_entry saved_posted_rx_list;
+
+	struct rxm_conn_close *close_data;
+};
+
+struct rxm_conn_close {
+	struct fid_ep *msg_ep;
+	struct dlist_entry posted_rx_list;
+};
+
+enum rxm_cmd_internal_type {
+	RXM_CMD_CONNREQ_ACCEPT,
+	RXM_CMD_CONNECTED,
+	RXM_CMD_CONN_CLOSE,
+	RXM_CMD_CONN_FREE,
+};
```

Comment:
Can this be merged with cmap cmd type? There isn't anything RXM specific except for the data which is generic already.

Reply:
Sure. I moved this into ofi_util.h